Update changelog-server.md with descriptions  of the 23-4 server version

katidisergei opened the pull request:
Update changelog-server.md with descriptions  of the 23-4 server version

### Changelog entry <!-- a user-readable short description of changes introduced in this PR -->
Update changelog-server.md with descriptions  of the 23-4 server version 

...

### Changelog category <!-- remove all except one -->
* Performance improvement
* Bugfix 

### Additional information

...

katidisergei committed:
Update changelog-server.md with descriptions  of the 23-4 server version 

Update changelog-server.md with descriptions  of the 23-4 server version

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+**Производительность:**
+
+* Исправлена проблема повышенного потребления CPU актором топиков `PERSQUEUE_PARTITION_ACTOR`.
+* [Оптимизировано](https://github.com/ydb-platform/ydb/pull/2083) использование ресурсов репликами Schemeboard. Особенно заметно при наличии изменений таблиц с большим числом партиций. 
```
Наибольший эффект заметен при модификации метаданных таблиц с большим количеством партиций.

katidisergei replied:
поменял

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+
+**Исправления ошибок:**
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым producer_id при выключенной дедупликации.
```
, и приводившая к аварийному завершению серверного процесса.

katidisergei replied:
добавил

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+**Исправления ошибок:**
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым producer_id при выключенной дедупликации.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных.
```
, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и аварийным завершением серверного процесса.

katidisergei replied:
добавил

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым producer_id при выключенной дедупликации.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка частичной фиксации накопленных изменений при использовании  распределенных транзакций.
```
[Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании  распределенных транзакций.

katidisergei replied:
поправил

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка частичной фиксации накопленных изменений при использовании  распределенных транзакций.
+* Исправлена ошибка отображения метрики количества партиций в топике.
+* Исправлена проблема, приводящая к падению процесса YDB из за ошибочного состояния сессии записи в топик.
+* Исправлены внутренние серверные ошибки BlobStorageController. 
```
* Исправлена ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных.
* Исправлена ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на узлы кластера.

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+* Исправлена проблема, приводящая к падению процесса YDB из за ошибочного состояния сессии записи в топик.
+* Исправлены внутренние серверные ошибки BlobStorageController. 
+* Исправлена ошибка  INTERNAL_ERROR (has already committed) в стриминговых запросах.
+* Исправлены утечки памяти в C++ SDK.
```
, которые в некоторых сценариях могли проявляться в серверных процессах.

blinkov replied:
тогда желательно вместо «в некоторых сценариях» более чётко написать в каких

zinal reviewed (changes requested):
Хотелось бы при описании "исправлена ошибка" видеть хотя бы в минимальном виде возможные ситуации проявления и эффекты этой ошибки. Без этой информации невозможно оценить вносимые правки.

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
 # Список изменений {{ ydb-short-name }} Server
 
+## Версия 23.3 {#23-4}
```
```suggestion
## Версия 23.4 {#23-4}
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым producer_id при выключенной дедупликации.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка частичной фиксации накопленных изменений при использовании  распределенных транзакций.
+* Исправлена ошибка отображения метрики количества партиций в топике.
```
из этого предложения не понятно: эта метрика совсем не отображалась или выдавала кривые значения?

katidisergei replied:
поправил

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка частичной фиксации накопленных изменений при использовании  распределенных транзакций.
+* Исправлена ошибка отображения метрики количества партиций в топике.
+* Исправлена проблема, приводящая к падению процесса YDB из за ошибочного состояния сессии записи в топик.
```
```suggestion
* Исправлена проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+* Исправлена ошибка отображения метрики количества партиций в топике.
+* Исправлена проблема, приводящая к падению процесса YDB из за ошибочного состояния сессии записи в топик.
+* Исправлены внутренние серверные ошибки BlobStorageController. 
+* Исправлена ошибка  INTERNAL_ERROR (has already committed) в стриминговых запросах.
```
```suggestion
* Исправлена ошибка `INTERNAL_ERROR (has already committed)` в стриминговых запросах.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+* Исправлена ошибка отображения метрики количества партиций в топике.
+* Исправлена проблема, приводящая к падению процесса YDB из за ошибочного состояния сессии записи в топик.
+* Исправлены внутренние серверные ошибки BlobStorageController. 
+* Исправлена ошибка  INTERNAL_ERROR (has already committed) в стриминговых запросах.
```
что тут имеется ввиду под стриминговыми запросами?

zinal replied:
Это YQ в режиме стриминга, оконные запросы над топиками. Напрямую в YDB эту функцию сейчас сложно задействовать (как минимум инструкций нет).

katidisergei replied:
не совсем понятно итого надо что то менять или нет, трогать не стал

s-sabitova replied:
Предлагаю здесь добавить ссылку на эту страницу про стриминговые запросы: https://ydb.tech/docs/ru/reference/ydb-cli/yql

katidisergei replied:
Добавил

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei committed:
Update changelog-server.md

fixed errors and inaccuracies in the text of the article

katidisergei committed:
Update changelog-server.md

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+
+**Исправления ошибок:**
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым producer_id при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
```
```suggestion
* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+
+**Производительность:**
+
+* Исправлена проблема повышенного потребления CPU актором топиков `PERSQUEUE_PARTITION_ACTOR`.
+* [Оптимизировано](https://github.com/ydb-platform/ydb/pull/2083) использование ресурсов репликами Schemeboard. Наибольший эффект заметен при модификации метаданных таблиц с большим количеством партиций.
```
```suggestion
* Исправлена проблема повышенного потребления вычислительных ресурсов актором топиков `PERSQUEUE_PARTITION_ACTOR`.
* [Оптимизировано](https://github.com/ydb-platform/ydb/pull/2083) использование ресурсов репликами SchemeBoard. Наибольший эффект заметен при модификации метаданных таблиц с большим количеством партиций.
```

blinkov committed:
Apply suggestions from code review

s-sabitova commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и аварийным завершением серверного процесса.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций.
+* Исправлена ошибка отображения метрики количества партиций в топике, количестко партиций отображалось некорректно.
```
Исправлена ошибка, из-за которой метрика количества партиций в топике могла иметь неверное значение

katidisergei committed:
Update changelog-server.md

I removed the typo in line 17
and removed the mention of the C++ SDK as it turned out, this mention refers to Logbroker

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и аварийным завершением серверного процесса.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций.
+* Исправлена ошибка отображения метрики количества партиций в топике, количество партиций отображалось некорректно.
```
```suggestion
* Исправлена ошибка отображения метрики количества партиций в топике, ранее количество партиций отображалось некорректно.
```

blinkov replied:
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3587) ошибка отображения метрики количества партиций в топике, ранее в ней отображалось некорректное значение.
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+**Исправления ошибок:**
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и аварийным завершением серверного процесса.
```
```suggestion
* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и аварийным завершением серверного процесса.
```

blinkov replied:
* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и, как следствие, аварийным завершением серверного процесса `ydbd`.

blinkov replied:
```suggestion
* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и, как следствие, аварийным завершением серверного процесса `ydbd`.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+* Исправлена проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
+* Исправлена ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных.
+* Исправлена ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на узлы кластера.
+* Исправлена ошибка  INTERNAL_ERROR (has already committed) в [стриминговых запросах](https://ydb.tech/docs/ru/reference/ydb-cli/yql).
```
саму ошибку в бэктики ещё

blinkov replied:
```suggestion
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций.
+* Исправлена ошибка отображения метрики количества партиций в топике, количество партиций отображалось некорректно.
+* Исправлена проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
+* Исправлена ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3002) ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных. Ошибка проявлялась для систем с явным образом включенной функцией "blob depot" (по умолчанию функция выключена).
```

blinkov replied:
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3002) ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных. Ошибка проявлялась для систем с явным образом включенной экспериментальной функцией "blob depot" (по умолчанию эта функция выключена).
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+* Исправлена ошибка отображения метрики количества партиций в топике, количество партиций отображалось некорректно.
+* Исправлена проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
+* Исправлена ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных.
+* Исправлена ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на узлы кластера.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2696) ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на отдельные узлы кластера. В результате в редких случаях могли блокироваться операции чтения и записи данных, хранящихся в затронутой группе, и требовалось ручное вмешательство администратора.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+
+**Исправления ошибок:**
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2475) ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации. Она могла приводить к аварийному завершению серверного процесса `ydbd`.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+
+* Исправлена ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации, и приводившая к аварийному завершению серверного процесса.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сжатия данных, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и аварийным завершением серверного процесса.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций. Данная ошибка возникает при крайне редкой комбинации событий, включающей в себя перезапуск таблеток, обслуживающих вовлеченные в транзакцию партиции таблиц.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,26 @@
+* Исправлена ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных.
+* Исправлена ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на узлы кластера.
+* Исправлена ошибка  INTERNAL_ERROR (has already committed) в [стриминговых запросах](https://ydb.tech/docs/ru/reference/ydb-cli/yql).
+* Исправлены ошибки, которые в том числе приводили утечкам памяти и рестартам узлов ydb topics.
```
```suggestion
* [Устранены](https://github.com/ydb-platform/ydb/pull/2126) утечки памяти, которые проявлялись при копировании данных топиков между кластерами {{ ydb-short-name }}. Они могли приводить к завершению серверных процессов `ydbd`  из-за исчерпания доступной оперативной памяти.
```

blinkov committed:
Apply suggestions from code review

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+
+**Исправления ошибок:**
+
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2475) ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации. Она могла приводить к аварийному завершению серверного процесса `ydbd`.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и, как следствие, аварийным завершением серверного процесса `ydbd`.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций. Данная ошибка возникает при крайне редкой комбинации событий, включающей в себя перезапуск таблеток, обслуживающих вовлеченные в транзакцию партиции таблиц.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3587) ошибка отображения метрики количества партиций в топике, ранее в ней отображалось некорректное значение.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2651) проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3002) ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных. Ошибка проявлялась для систем с явным образом включенной экспериментальной функцией "blob depot" (по умолчанию эта функция выключена).
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2696) ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на отдельные узлы кластера. В результате в редких случаях могли блокироваться операции чтения и записи данных, хранящихся в затронутой группе, и требовалось ручное вмешательство администратора.
+* [Устранены](https://github.com/ydb-platform/ydb/pull/2126) утечки памяти, которые проявлялись при копировании данных топиков между кластерами {{ ydb-short-name }}. Они могли приводить к завершению серверных процессов `ydbd`  из-за исчерпания доступной оперативной памяти.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций. Данная ошибка возникает при крайне редкой комбинации событий, включающей в себя перезапуск таблеток, обслуживающих вовлеченные в транзакцию партиции таблиц.
* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и, как следствие, аварийным завершением серверного процесса `ydbd`.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2696) ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на отдельные узлы кластера. В результате в редких случаях могли блокироваться операции чтения и записи данных, хранящихся в затронутой группе, и требовалось ручное вмешательство администратора.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3002) ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных. Ошибка проявлялась для систем с явным образом включенной экспериментальной функцией "blob depot" (по умолчанию эта функция выключена).
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2475) ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации. Она могла приводить к аварийному завершению серверного процесса `ydbd`.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2651) проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3587) ошибка отображения метрики количества партиций в топике, ранее в ней отображалось некорректное значение.
* [Устранены](https://github.com/ydb-platform/ydb/pull/2126) утечки памяти, которые проявлялись при копировании данных топиков между кластерами {{ ydb-short-name }}. Они могли приводить к завершению серверных процессов `ydbd`  из-за исчерпания доступной оперативной памяти.
```

blinkov committed:
Update ydb/docs/ru/core/changelog-server.md

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,25 @@
+
+**Исправления ошибок:**
+
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций. Данная ошибка возникает при крайне редкой комбинации событий, включающей в себя перезапуск таблеток, обслуживающих вовлеченные в транзакцию партиции таблиц.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и, как следствие, аварийным завершением серверного процесса `ydbd`.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2696) ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на отдельные узлы кластера. В результате в редких случаях могли блокироваться операции чтения и записи данных, хранящихся в затронутой группе, и требовалось ручное вмешательство администратора.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3002) ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных. Ошибка проявлялась для систем с явным образом включенной экспериментальной функцией "blob depot" (по умолчанию эта функция выключена).
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2475) ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации. Она могла приводить к аварийному завершению серверного процесса `ydbd`.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2651) проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3587) ошибка отображения метрики количества партиций в топике, ранее в ней отображалось некорректное значение.
+* [Устранены](https://github.com/ydb-platform/ydb/pull/2126) утечки памяти, которые проявлялись при копировании данных топиков между кластерами {{ ydb-short-name }}. Они могли приводить к завершению серверных процессов `ydbd`  из-за исчерпания доступной оперативной памяти.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка возможной неполной фиксации накопленных изменений при использовании распределенных транзакций. Данная ошибка возникает при крайне редкой комбинации событий, включающей в себя перезапуск таблеток, обслуживающих вовлеченные в транзакцию партиции таблиц.
* [Устранена](https://github.com/ydb-platform/ydb/pull/3165) гонка между процессами слияния таблиц и сборки мусора, из-за которой сборка мусора могла завершиться ошибкой нарушения инвариантов и, как следствие, аварийным завершением серверного процесса `ydbd`.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2696) ошибка в Blob Storage, из-за которой информация о смене состава группы хранения могла не поступать своевременно на отдельные узлы кластера. В результате в редких случаях могли блокироваться операции чтения и записи данных, хранящихся в затронутой группе, и требовалось ручное вмешательство администратора.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3002) ошибка в Blob Storage, из-за которой при корректной конфигурации могли не запускаться узлы хранения данных. Ошибка проявлялась для систем с явным образом включенной экспериментальной функцией "blob depot" (по умолчанию эта функция выключена).
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2475) ошибка, возникавшая в некоторых ситуациях записи в топик с пустым `producer_id` при выключенной дедупликации. Она могла приводить к аварийному завершению серверного процесса `ydbd`.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2651) проблема, приводящая к падению процесса `ydbd` из-за ошибочного состояния сессии записи в топик.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3587) ошибка отображения метрики количества партиций в топике, ранее в ней отображалось некорректное значение.
* [Устранены](https://github.com/ydb-platform/ydb/pull/2126) утечки памяти, которые проявлялись при копировании данных топиков между кластерами {{ ydb-short-name }}. Они могли приводить к завершению серверных процессов `ydbd` из-за исчерпания доступной оперативной памяти.
```

blinkov committed:
Update ydb/docs/ru/core/changelog-server.md
